src/main/java/com/projectkorra/rpg/storage/TableCreator.java

```diff
@@ -111,10 +111,10 @@ private void createRpgAvatarTable() {
 
     private void createRpgPastLivesTable() {
```

Comment:
These new tables created here in TableCreator differ from what AvatarManager is expecting given its methods.  For example, there are queries expecting a column "startTime" in the RPG_AVATAR_TABLE which is not included in your new table.

Reply:
Update: I'm seeing now that there are 2 avatar managers included here... We should only have 1 AvatarManager and should not add 2 of them to upstream. I'll provide some notes on that on that file